sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ConstantColumnVector.java

```diff
@@ -77,6 +77,11 @@ public ConstantColumnVector(int numRows, DataType type) {
     }
   }
 
+  public void closeIfFreeable() {
```

Comment:
`closeIfNotWritable`?

Reply:
ah yeah my apologies - do you think `closeIfNotWritable` still makes sense given that the `ConstantColumnVector` is not a `WritableColumnVector`? 

Do you think `closeIfFreeable` makes sense? If so, I'll change the method name in `ColumnVector.java` in this PR as well. 

Or do you have a preference for a different name?